src/foundry/prosemirror/schema/schema-definition.d.mts

```diff
@@ -0,0 +1,39 @@
+import { MarkSpec, Node, NodeSpec } from "prosemirror-model";
+
+export default SchemaDefinition;
+
+/**
+ * An abstract interface for a ProseMirror schema definition
+ */
+declare abstract class SchemaDefinition {
+  /**
+   * The HTML tag selector this node is associated with
+   */
+  static tag: string;
+
+  /**
+   * Schema attributes
+   * @abstract
```

Comment:
it's an inline property like `static` or `private`, not a tag. So in this case, `abstract static get attrs()`